### ISSUE TITLE
[Request]: Status Checker

### ISSUE THREAD

**Issue by Kolhax** [YimMenu/YimMenu#2583]
### Problem

Peoples dont know when to update the munu and get bans,...

### Solution

you should make some get requests to a version.txt file or smth like this to let the user know on injection if the menu is outdated or not?

### Reason

Peoples dont know when to update the munu and get bans,...

**Comment by Yimura** [comment]
You're right on wanting us to add such a thing, I've actually been thinking about adding some stuff to YimMenu to make sure that whenever it gets detected users are made aware of it.

1. Show a non-intrustive message that their YimMenu is outdated and they should get a new version
2. Show an intrusive message that needs to contains 2 actions [Unload] and [Continue Anyways] whenever YimMenu is detected

**Comment by gir489returns** [comment]
Should probably make it read a file on the GitHub page; could probably slip it in when we add the system to pull Lua files from the repo.

**Comment by pierrelasse** [comment]
That's a good idea. Would it be possible to add a option to automaticly update and replace the current dll?

**Comment by Kolhax** [comment]
Hey, nothing big in it self but I have made an python script for my self for auto update if yall need I can drop it here, it has less then 20 lines I think, it ping github searching in the release if the file is the same if not, it download the new dll and xenos, you can make profiles .xpr that auto inject to à profile, so I update the dll and run the xpr and it works fine

**Comment by gir489returns** [comment]
We have plans to make a system within YimMenu.dll warn the user when they go too far out of date (even so much as to potentially read flags like X commit is detected, hard warn about it). But those will come with the Lua change that we have planned.

For right now, development has slowed basically to quality-of-life fixes and very minor feature additions.

**Comment by Kolhax** [comment]
glad to hear this
